src/components/submission/feed-entry.vue

```diff
@@ -157,11 +160,9 @@ export default {
   },
   methods: {
```

Comment:
if you are up for it then you can get rid of all these function by using optional chaining